ci/scripts/jenkins/test.sh

```diff
@@ -19,49 +19,26 @@ set -e
 source ${WORKSPACE}/ci/scripts/jenkins/common.sh
 /usr/bin/nvidia-smi
 
-gpuci_logger "Check versions"
-python3 --version
-gcc --version
-g++ --version
+restore_conda_env
 
-gpuci_logger "Check conda environment"
-conda info
-conda config --show-sources
-conda list --show-channel-urls
-
-gpuci_logger "Downloading build artifacts from ${DISPLAY_ARTIFACT_URL}"
-aws s3 cp --no-progress "${ARTIFACT_URL}/conda_env.tar.gz" "${WORKSPACE_TMP}/conda_env.tar.gz"
-aws s3 cp --no-progress "${ARTIFACT_URL}/workspace.tar.bz" "${WORKSPACE_TMP}/workspace.tar.bz"
-
-gpuci_logger "Extracting"
-mkdir -p /opt/conda/envs/morpheus
-tar xf "${WORKSPACE_TMP}/conda_env.tar.gz" --no-same-owner --directory /opt/conda/envs/morpheus
-tar xf "${WORKSPACE_TMP}/workspace.tar.bz" --no-same-owner
-
-gpuci_logger "Setting test env"
-conda activate morpheus
-conda-unpack
-conda list --show-channel-urls
-
-npm install --slient -g camouflage-server
+npm install --silent -g camouflage-server
 mamba install -q -y -c conda-forge "git-lfs=3.1.4"
 
 gpuci_logger "Pulling LFS assets"
 cd ${MORPHEUS_ROOT}
+
 git lfs install
 git lfs pull
 
-pip install -e ${MORPHEUS_ROOT}
-
-# Work-around for issue where libmorpheus_utils.so is not found by libmorpheus.so
-# The build and test nodes have different workspace paths (/jenkins vs. /var/lib/jenkins)
-# Typically these are fixed by conda-unpack but since we did an in-place build we will
-# have to fix this ourselves by setting LD_LIBRARY_PATH
-export LD_LIBRARY_PATH=${LD_LIBRARY_PATH}:${MORPHEUS_ROOT}/morpheus/_lib
-
 gpuci_logger "Running tests"
+# Running the tests from the tests dir. Normally this isn't nescesary, however since
+# we are testing the installed version of morpheus in site-packages and not the one
+# in the repo dir, the pytest coverage module reports incorrect coverage stats.
+# Note: The -I flag prevents python from automatically adding the current dir to path,
+# so this bug appears to be specific to the coverage plugin
+cd ${MORPHEUS_ROOT}/tests
 set +e
-pytest --run_slow \
+python -I -m pytest --run_slow \
```

Comment:
Couple of questions about this:

1. What is the `-I` flag needed for?
2. Have you looked into the import options in pytest to help fix this issue?